lib/modules/platform/github/index.ts

```diff
@@ -166,14 +170,26 @@ export async function initPlatform({
     }
   }
   logger.debug({ platformConfig, renovateUsername }, 'Platform config');
-  const platformResult: PlatformResult = {
+  const gitHubRule: HostRule[] = [];
+  if (!platformConfig.isGhe) {
+    gitHubRule.push({
+      matchHost: 'ghcr.io',
+      username: renovateUsername,
+      password: token,
+    });
+    gitHubRule.push({
+      matchHost: '.pkg.github.com',
```

Comment:
```suggestion
      matchHost: 'pkg.github.com',
```

Reply:
i think this should only match subdomains, not `pkg.github.com` directly. not sure if this is a problem

Reply:
just for your convenience, it's the same result